packages/richtext-lexical/src/features/blocks/client/component/index.tsx

```diff
@@ -197,6 +203,7 @@ export const BlockComponent: React.FC<Props> = (props) => {
         },
         docPreferences: await getDocPreferences(),
         formState: prevFormState,
+        fullFormStateOverride: deepCopyObjectSimpleWithoutReactComponents(parentDocumentFields),
```

Comment:
I also do not love this name. Feels like this could be called `parentDocFormState` or similar. Or `topLevelDoc`?